FEXCore/Scripts/config_generator.py

```diff
@@ -148,9 +148,15 @@ def print_man_options(options):
             if (value_type == "strenum"):
                 Enums = op_vals["Enums"]
                 output_man.write("\\fBAvailable Options:\\fR\n")
+                item_count = len(Enums.items())
+                item_num = 0
                 for enum_op_key, enum_op_vals in Enums.items():
-                    output_man.write("{}, ".format(enum_op_vals))
-                output_man.write("\n")
+                    output_man.write("{}".format(enum_op_vals))
+                    item_num = item_num + 1
+                    if item_num != item_count:
+                        output_man.write(", ")
```

Comment:
Combining f-strings with `join()` expresses this much more succinctly without manual state tracking:
```suggestion
                output_man.write(", ".join(f"{enum_op_val}" for [_, enum_op_val] in Enums.items()))
```

Reply:
Neat, switched over to that.